src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs

```diff
@@ -256,6 +256,19 @@ public async Task<IActionResult> ChangePassword(ChangePasswordViewModel model, s
     public IActionResult ChangePasswordConfirmation()
         => View();
 
+    /// <summary>
+    /// This action is retained for backward compatibility.
+    /// Both this action and <see cref="ExternalAuthenticationsStartupFilter"/> are scheduled for removal in version 3.
+    /// </summary>
+    /// <returns></returns>
+    [HttpPost]
+    [AllowAnonymous]
+    [ValidateAntiForgeryToken]
+    public IActionResult ExternalLogin()
```

Comment:
Apart from the `ExternalLogin` action, there was also ``ExternalLogin` and `DefaultExternalLogin` that one could imagine referencing from custom code. Shouldn't those be taken care of too?

Reply:
I don't think `DefaultExternalLogin` is used by anyviews. This action is redirected to from the AccountController.Login which is still the case. So I don't think this is needed unless you think it is used in views that I am not aware of.

Reply:
Not just views, but any reference, including standard links. But looking into it, these don't seem to be affected.